src/pages/Regions/Region.tsx

```diff
@@ -431,7 +431,7 @@ const Region = () => {
   };
 
   const getNineMembers = (member: any[], amount: number) => {
-    if (member.length > 6) {
+    if (member.length > 9) {
```

Comment:
magic number 9. Please, create a const variable for this number (something like "maxMembersDisplayCount")

Reply:
Done